Mission involves creep.lua

```diff
@@ -0,0 +1,86 @@
+local argsUtil = require("Module:ArgsUtil")
+local cargoUtil = require("Module:CargoUtil")
+
+local p = {}
+
+-- kill ranges based on difficulty level
+local killRanges = {
```

Comment:
even though it says "local", because this is scoped outside of any function, it's technically a global variable. Global variables are often bad but they can be ok! Especially if they are constant! This one seems like a constant so it's ok

usually as a convention, in all programming languages, people write `local KILL_RANGES =` with all caps and underscores like this. That way you know that it's not safe/allowed to modify this variable because it's probably global